utils/config.py

```diff
@@ -0,0 +1,5 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+DATASET = os.getenv("DATASET")
```

Comment:
It is better to use `os.environ[]`. It raises an error in case of absence of the env var. `os.getenv()` will return `None`.